examples/cluster-component/componentplan.yaml

```diff
@@ -10,7 +10,7 @@ spec:
   override:
     set:
     # according to https://kubebb.github.io/website/docs/core/get_started#kind%E5%BC%80%E5%8F%91%E9%9B%86%E7%BE%A4
-    - ingress-nginx.deployedHost: kubebb-core-control-plane
+    - ingress-nginx.deployedHost=kubebb-core-control-plane
```

Comment:
**The override part should be written as**
```yaml
  override:
    set:
    # according to https://kubebb.github.io/website/docs/core/get_started#kind%E5%BC%80%E5%8F%91%E9%9B%86%E7%BE%A4
    - ingress-nginx.controller.nodeSelector.kubernetes\.io/hostname=kubebb-core-worker
```

**Result**

```shell
(base) ➜  cluster-component git:(main) ✗ kubectl get po -nkubebb-system -owide
NAME                                                          READY   STATUS    RESTARTS   AGE     IP           NODE                  NOMINATED NODE   READINESS GATES
cert-manager-56585488fd-kc9d9                                 1/1     Running   0          7m15s   10.244.1.5   kubebb-core-worker    <none>           <none>
cert-manager-cainjector-797b96c765-ccrhn                      1/1     Running   0          7m15s   10.244.2.6   kubebb-core-worker2   <none>           <none>
cert-manager-webhook-95fbbccd8-wfskx                          1/1     Running   0          7m15s   10.244.1.6   kubebb-core-worker    <none>           <none>
cluster-component-ingress-nginx-controller-78b599568d-bnfxn   1/1     Running   0          2m20s   10.244.1.7   kubebb-core-worker    <none>           <none>
```
---

**Reason**
There's a problem here that we need to deal with, the source yaml is:
```yaml
ingress-nginx:
  deployedHost: &deployedHost
    kind-worker
  ...
  controller:
   nodeSelector
     kubernetes.io/os: linux
     kubernetes.io/hostname: *deployedHost
```

if we execute
```shell
helm install cluster-component --set ingress-nginx.deployedHost="some-node" .
```

First we will get the following yaml (yaml expand reference field)
```yaml
ingress-nginx:
  deployedHost:  kind-worker
  ...
  controller:
   nodeSelector
     kubernetes.io/os: linux
     kubernetes.io/hostname: kind-worker
```

And then, rewrite `ingress-nginx.deployedHost`, we will get the following yaml

```yaml
ingress-nginx:
  deployedHost:  some-node
  ...
  controller:
   nodeSelector
     kubernetes.io/os: linux
     kubernetes.io/hostname: kind-worker
```

Therefore, even if we rewrite the ingress-nginx.deployedHost field in the componentplan, the Pod is still scheduled to the kind-worker node.